doc/Building Tribler on Windows.md

```diff
@@ -24,8 +24,10 @@ Inside the `build` folder, put the following items:
 1. A static version (64 bit, git-1d8f9b7) of ffmpeg, available [here](http://ffmpeg.zeranoe.com/builds/). Place it in a folder called `ffmpeg` in the `build` folder.
 2. A folder `certs` containing a `.pfx` key. In our case it's named `swarmplayerprivatekey.pfx`. Make sure to rename paths in `makedist_win.bat` to match your file name.
 3. A `vlc` folder containing a full instalation of vlc (Version 2.2.1).
-4. `vc_redist.exe`, which is available [here](https://www.microsoft.com/en-us/download/details.aspx?id=48145). Make sure to choose the 64 bit one! (In case you build 32 bit, get the x86 version). Don't forget to rename.
-5. `libsodium.dll` which can be downloaded from [libsodium.org](https://download.libsodium.org/libsodium/releases/) (as of writing version 1.0.8).
+4. `vc_redist_90.exe` (Microsoft Visual C++ 2008 Redistributable Package), which is available [here](https://www.microsoft.com/en-us/download/details.aspx?id=15336). Make sure to choose the 64 bit one! (In case you build 32 bit, get the x86 version [here](https://www.microsoft.com/en-us/download/details.aspx?id=29). Don't forget to rename the file.
```

Comment:
I would remove the (In case you build 32 bit...) out of the parents and change the sentence to:

In case you are building a 32-bit version of Tribler, download the package from here

or something similar.